documentation/modules/configuring/con-config-high-volume-messages.adoc

```diff
@@ -0,0 +1,214 @@
+
+[role="_abstract"]
+Operations like tracking web activity can generate a high volume of messages. 
+But even a data source with moderate throughput can produce a high volume of messages if it has a lot of existing data. 
+If your Strimzi deployment needs to handle a high volume of messages, you can use configuration options to optimize for throughput and latency. 
+
+Producer and consumer configuration helps to control the size and frequency of requests to Kafka brokers. 
+For more information on the configuration options, see the following:
+
+* xref:con-producer-config-properties-throughput-{context}[] 
+* xref:con-consumer-config-properties-throughput-{context}[]  
+
+You can also use the same configuration options with the Kafka Connect runtime producers and consumers used by Kafka Connect and MirrorMaker 2.0 source connectors.
+Consumers are used by source connectors to retrieve data from an external data system for Kafka Connect or from a source Kafka cluster for MirrorMaker 2.0.
+Producers are used by source connectors to send messages to a target Kafka cluster.
+
+For consumers, you might increase the amount of data fetched in a single fetch request to reduce latency.
+You increase the fetch request size using the `fetch.max.bytes` and `max.partition.fetch.bytes` properties.
+You can also set a maximum limit on the number of messages returned from the consumer buffer using the `max.poll.records` property.
+
+Configure the `fetch.max.bytes`, `max.partition.fetch.bytes`, and `max.poll.records` values at the source connector level (`consumer.*`), as they relate to the specific consumer that fetches messages from the source.
+
+For producers, you might increase the size of the message batches sent in a single produce request.
+You increase the batch size using the `batch.size` property.
+A larger batch size reduces the number of outstanding messages ready to be sent and the size of the backlog in the message queue.
+Messages being sent to the same partition are batched together.
+A produce request is sent to the target cluster when the batch size is reached.
+By increasing the batch size, produce requests are delayed and more messages are added to the batch and sent to brokers at the same time.  
+This can improve throughput when you have just a few topic partitions that handle large numbers of messages.  
+
+Consider the number and size of the records that the producer handles for a suitable producer batch size. 
+
+Use `linger.ms` to add a wait time in milliseconds to delay produce requests when producer load decreases. 
+The delay means that more records can be added to batches if they are under the maximum batch size.  
+
+Configure the `batch.size` and `linger.ms` values at the source connector level (`producer.override.*`), as they relate to the specific producer that sends messages to the target Kafka cluster.
+
+For Kafka Connect, the data replication pipeline to the target Kafka cluster is as follows:
```

Comment:
I would avoid the term replication ... maybe ingestion?

Reply:
I thought _streaming_ might be clearer.